challenge/package.json

```diff
@@ -22,7 +22,7 @@
         "react-scripts": "^4.0.1"
     },
     "scripts": {
-        "start": "react-scripts start",
+        "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
what is this change for?

Reply:
I believe during the interview I asked you and Serena about why my npm start was not working. You guys suggested to add this and it fixed the problem. I believe it had something to do with my node version.

Reply:
Oh right, can you leave that out of the commit. That may cause errors for other people.

Reply:
I added a new commit that removes the information that I added. I hope thats okay. Thank you.